### PR TITLE
fix deploy script (SDKCF-2399)

### PR DESCRIPTION
**Pull request by imaginaris**
# Description
Script deploy provider doesn't accept an array of commands

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [X] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data **before every commit**, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,10 +24,10 @@ deploy:
   # When tagged with version (X.X.X)
   # Deploy Pod spec
   - provider: script
-    cleanup: false
+    edge: true
     script:
-      - bundle exec pod spec lint --allow-warnings
-      - bundle exec pod trunk push --allow-warnings --verbose
+      bundle exec pod spec lint --allow-warnings;
+      bundle exec pod trunk push --allow-warnings --verbose
```
Comment (donnie-jp) [inline .travis.yml:30]: Maybe we can just put it in a `scripts/deploy-pod.sh` file instead of this workaround?

File: .travis.yml [hunk 2]
```diff
@@ -24,10 +24,10 @@ deploy:
   # When tagged with version (X.X.X)
   # Deploy Pod spec
   - provider: script
-    cleanup: false
+    edge: true
```
Comment (donnie-jp) [inline .travis.yml:27]: Why do we want `edge`?

File: .travis.yml (hunk 1 above)
Comment (imaginaris) [reply .travis.yml:30]: We could. But this way seems more transparent and it's just 2 commands.

File: .travis.yml (hunk 2 above)
Comment (imaginaris) [reply .travis.yml:27]: Mostly because of this feature:
`Failed deployments now always error the build when using dpl v2`

File: .travis.yml (hunk 1 above)
Comment (donnie-jp) [reply .travis.yml:30]: ok

**Pull request merged**